docs/camino-node/auto-upgrade.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 6
+title: Auto Upgrade Node
+description: Automatically Upgrade Camino Node
+---
+
+# Automatically Upgrade Camino Node
+
+This guide provides instructions on how to automatically upgrade camino-node using a bash script and crontab.
+
+## Automatic Install Script
+
+If you have previously installed camino-node using the autoinstall script, you can now configure an automatic
+upgrade with the following steps:
+
+### Download Node Upgrader
+
+First, download the node upgrader script. You can place this script anywhere on the node's operating system.
+For the sake of this guide, we will assume you put it under the `/root` directory.
+
+```
+wget -nd -m https://raw.githubusercontent.com/chain4travel/camino-docs/c4t/scripts/camino-node-upgrader.sh;\
```

Comment:
are you going to add the script in camino-docs or different repo ?

Reply:
I've added it to docs repo for now. We can move it later if we create another repo for utility scripts.